docs/src/general_configuration/boxes.md

```diff
@@ -39,15 +39,26 @@ Box description files are located in `~/.config/mdbci/boxes` folder or in `confi
     "box": "generic/debian11",
     "platform": "debian",
     "platform_version": "bullseye"
-  }
+  },
+
+  "rhel_9_ibm": {
```

Comment:
Please split examples into a set of JSON documents. MDBCI can not support different providers in one template.